### PR TITLE
Add Get to team package

### PR DESCRIPTION
**Pull request by patrickxb**

### CODE REVIEW

**Comment by maxtaco** [comment]
high level question: should this be an engine? seems like lib should be a better place for it.

Second question: with @mlsteele, let's move all teams code into its own non-libkb lib?

**Comment by maxtaco** [comment]
In other words, it's likely not an engine since I think it's going to be called a lot as a utility as part of what other engines are doing.

**Comment by patrickxb** [comment]
I thought the ticket had "engine" in it, but I guess it doesn't.

**Comment by maxtaco** [comment]
Yeah I realized after the fact about that, but I meant it as shorthand for "engine and below"

File: protocol/avdl/keybase1/teams.avdl [hunk 1]
```diff
@@ -2,4 +2,5 @@
 
 protocol teams {
   void teamCreate(int sessionID, string name);
+  void teamGet(int sessionID, string name);
```
Comment (maxtaco) [inline protocol/avdl/keybase1/teams.avdl]: do we still need this?

Comment (patrickxb) [inline protocol/avdl/keybase1/teams.avdl]: The CLI will need it to run `keybase team list-memberships nike`

Comment (patrickxb) [inline protocol/avdl/keybase1/teams.avdl]: so I left it there, even though it doesn't return anything useful...

**Comment by maxtaco** [comment]
ok great thanks 👍

**Comment by patrickxb** [comment]
c/i failure was osx timeout.  windows, linux passed + circle ci.  ran osx tests locally, passed.  so merging.

**Pull request merged**